code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -264,3 +302,75 @@
 	if(!cultie)
 		return FALSE
 	return cultie.total_sacrifices >= target_amount
+
+/datum/objective/ascend
+	name = "ascend"
+
+/datum/objective/ascend/update_explanation_text()
+	. = ..()
+	explanation_text = "Applease the Gods and ascend."
+
+/datum/objective/ascend/check_completion()
+	if(!owner)
+		return FALSE
+	var/datum/antagonist/heretic/cultie = owner.has_antag_datum(/datum/antagonist/heretic)
+	if(!cultie)
+		return FALSE
+	return cultie.ascended
+
+/datum/objective/minicult
+	name = "mini cult"
+
+/datum/objective/minicult/update_explanation_text()
+	. = ..()
+	target_amount = rand(2,5)
```

Comment:
don't randomise the target amount in update_explanation_text

Reply:
Uh oh! Sacrifice also does that!